ush/job_preamble.sh

```diff
@@ -42,10 +42,11 @@ export SENDWEB="${SENDWEB:-${SENDWEB_dfv}}"
 export KEEPDATA="${KEEPDATA:-${KEEPDATA_dfv}}"
 export MAILTO="${MAILTO:-${MAILTO_dfv}}"
 export MAILCC="${MAILCC:-${MAILCC_dfv}}"
+export PDYm1=$( $DATE_UTIL --date "${PDY} -1 day" "+%Y%m%d" )
```

Comment:
@JianpingHuang-NOAA, `PDYm1` is defined by running `. ./PDY` of `prod_util` (see the line 133 in this file). Please check this again after removing this line because this is defined in Line 133. This kind of definition is not valid for the `nco` mode.